provelo_custom/views/hr_kanban_views_employees.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="hr_kanban_view_employee_extend" model="ir.ui.view">
+        <field name="name">hr_kanban_view_phone_mail</field>
+        <field name="inherit_id" ref="hr.hr_kanban_view_employees" />
+        <field name="model">hr.employee</field>
+        <field name="arch" type="xml">
+            <field name="work_location" position="after">
+                <li t-if="record.work_email.raw_value">
+                    <i class="fa fa-envelope" />
+                    <field name="work_email" />
+                </li>
+                <li t-if="record.work_phone.raw_value">
+                    <i class="fa fa-phone" />
+                    <field name="work_phone" />
+                </li>
+                <li t-if="record.mobile_phone.raw_value">
+                    <i class="fa fa-mobile" />
+                    <field name="mobile_phone" />
+                </li>
+            </field>
+            <xpath expr='//span[contains(@class, "oe_hr_attendance_status_red")]' position="attributes">
```

Comment:
- not good: `@class='xxx'`
- better: `contains(@class, 'xxx')` 👏 
- best `hasclass('xxx')`

Reply:
Idem, commit has been removed